bower.json

```diff
@@ -17,6 +17,10 @@
     "package.json"
   ],
   "dependencies": {
-    "purescript-prelude": "^3.0.0"
+    "purescript-prelude": "^3.0.0",
+    "purescript-foldable-traversable": "^3.3.0"
```

Comment:
Another thought on `traverseRec`: separate typeclasses for it. That makes the change un/less breaking.